tests/core/uboot-unpacked-assets/task.yaml

```diff
@@ -1,15 +1,23 @@
 summary: Ensure we have unpacked kernel.img/initrd.img on uboot systems
 
-# TODO:UC20: enable for uc18/uc20 arm when that is ready
-systems: [ubuntu-core-16-arm-*]
+systems: [ubuntu-core-*-arm-*]
 
 environment:
     NAME/initrdimg: initrd.img*
     NAME/kernelimg: kernel.img*
 
 execute: |
-    output=$(find /boot/uboot/*-kernel_*.snap/ -name "$NAME" )
-    if [ -z "$output" ]; then
-        echo "Not found expected file $NAME in /boot/uboot/*-kernel_*.snap/"
-        exit 1
+    if os.query is-core20; then
+        # on UC20, the kernel snap is extracted onto ubuntu-seed, not on ubuntu-boot
+        output=$(find /run/mnt/ubuntu-seed/systems/*/kernel/*-kernel_*.snap/ -name "$NAME" )
```

Comment:
is this path correct for arm devices? Could be on /run/mnt/ubuntu-seed/snaps instead?

Reply:
There are kernel snap files under /snaps but the kernel snap is not extracted there

Reply:
ah, ok, thanks for the answer